lamin_cli/__main__.py

```diff
@@ -99,7 +99,7 @@ def wrapper(*args, **kwargs):
 @lamin_group_decorator
 @click.version_option(version=lamindb_version, prog_name="lamindb")
 def main():
-    """Configure LaminDB and perform simple actions."""
+    """CLI for LaminDB configuration and basic operations."""
```

Comment:
Miiiight break style but I think it's okay.

Reply:
Yeah, I think it'd be good to consistently use a verbal style across the whole docs page.

But I still think this is an improvement to the previous docstring, so let's just keep it and wait if we can come up with something more consistent and as good content wise in the future.